app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,21 @@
+class SessionsController < ApplicationController
+
+  def new
+  end
+
+  def create
+    customer = Customer.find_by_email(params[:email])
+    if customer == customer.authenticate(params[:password])
```

Comment:
oh wait haha ok nevermind
it allowed me to login if i use the correct password and redirected me if it was wrong
yea forget everything i just said....

== should be &&